lutris/installer/commands.py

```diff
@@ -292,6 +292,26 @@ def move(self, params):
             # Change game file reference so it can be used as executable
             self.game_files['src'] = src
 
+    def package(self, params):
+        """install a package on the preffered package manager"""
+        import os.path
+        # get the sytem package manager
+        portageInstalled = os.path.isdir("/usr/share/portage")
+        if portageInstalled:
+            packageManagerToUse = "portage"
+        else:
+            raise ScriptingError("No compatible system manager found on your system. actually, only portage work.")
+        # get the package id
+        package = params.get(packageManagerToUse)
+        if type(package) != str:
+            raise ScriptingError("no package specified for the '"+packageManagerToUse+"' package manager")
+        # install via the package manager
+        if packageManagerToUse == "portage":# for gentoo/linux, using emerge
+            return self.execute("kdesu -t -c 'emerge "+package+" --nospinner'")
```

Comment:
don't use kdesu.

Reply:
I didn't know by what I can replace this. I've solved the other issue in my local version.